src/components/SecondaryPanes/CommandBar.css

```diff
@@ -72,7 +72,7 @@ img.resume {
   border-bottom: none;
   background-color: var(--theme-body-background);
   border-top: 1px solid var(--theme-splitter-color);
-  flex: 0 0 26px;
+  flex: 0 0 25px;
```

Comment:
Showing a before and after view would be great, or maybe linking to the issue this solves - currently it says #none 
by introducing these pieces of information we have a chance of understanding the solution and what the problem was in the first place.

Reply:
edit: 
-added before and after
-their was no issue to reference to